### ISSUE TITLE
The edit/save-buttons for saved-searches are not reached from tabbing

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4195]
**Describe the bug**
The "Edit", "Save search" and "Update saved search" buttons in the filter-dropdown is not reached from the keyboard

**Where was the bug observed**
.io

**To Reproduce**
1. Tab to the filter-dropdown in data-view or map-view
2. Add a number of filters
3. Try reaching the "Save search" button from the keyboard, it is not working
4. Select a saved search in the dropdown, try reaching the "Edit" button from the keyboard, its not working
5. Change a number of filters and try reaching the "Update saved search" button from the keyboard, its not working

**Expected behavior**
All buttons should be reachable from the keyboard

**Is there a workaround? What is it.**
Only through clicking

**Screenshots**
<img width="833" alt="Screenshot 2021-01-21 at 19 20 30" src="https://user-images.githubusercontent.com/8624777/105394927-782e3300-5c1e-11eb-9cfa-c841d7be3a07.png">
<img width="767" alt="Screenshot 2021-01-21 at 19 25 33" src="https://user-images.githubusercontent.com/8624777/105394942-7c5a5080-5c1e-11eb-8627-6f8190d65a50.png">
<img width="600" alt="Screenshot 2021-01-21 at 19 20 22" src="https://user-images.githubusercontent.com/8624777/105394954-80866e00-5c1e-11eb-8f90-05c4964d989e.png">

**Comment by Angamanga** [comment]
The save/update saved search is fixed, edit-button is still not reachable.

**Comment by tuxpiper** [comment]
Fixed by ushahidi/platform-client#1652

**Comment by tuxpiper** [comment]
@Angamanga @Obadha2  has this been fixed for good? Can it be closed?

**Comment by Angamanga** [comment]
@tuxpiper Partly done, the "edit button" for saved searches is still not reachable.

**Comment by tuxpiper** [comment]
@Angamanga to take another look and assess complexity . Can we get it done in 1-2 weeks from today?
